changelog/7259.deprecation.rst

```diff
@@ -1,3 +1,12 @@
 ``py.path.local`` arguments for hooks have been deprecated. See :ref:`the deprecation note <legacy-path-hooks-deprecated>` for full details.
 
 ``py.path.local`` arguments to Node constructors have been deprecated. See :ref:`the deprecation note <node-ctor-fspath-deprecation>` for full details.
+
+.. note::
+    The name of the :class:`~_pytest.nodes.Node` arguments and attributes (the
+    new attribute being ``path``) is **the opposite** of the situation for hooks
```

Comment:
Not sure this still applies, as we don't have the same situation anymore.

Reply:
Well, kind of - we still have the situation that `Node.path` is new, but `path` in hooks is old. But if people prefer, I'm fine with removing those notes.

Reply:
Ahh you are correct. Nevermind, I misunderstood the text. 👍

Reply:
If you have a suggestion for a rewording to make things clearer, let me know! Probably going to be Monday or Tuesday until I can finish the release anyways.